.github/workflows/deploy.yml

```diff
@@ -63,7 +74,6 @@ jobs:
       - name: Set Environment variable
         run: |
           echo "DOCKER_IMAGE=$DOCKER_IMAGE" >> $GITHUB_ENV
-          if [ "${{ matrix.environment }}" = "production" ]; then echo "SPACE_SUFFIX=prod" >> $GITHUB_ENV; fi;
```

Comment:
why did we remove the `SPACE_SUFFIX` from being added here?

Reply:
It's a leftover from a previous refactor, SPACE_SUFFIX is not used anymore. I'll move it to a separate commit for clarity.